content/spin/v2/quickstart.md

```diff
@@ -282,21 +279,23 @@ The additional `spin.toml` file is the manifest file, which tells Spin what even
 <!-- @nocpy -->
 
 ```toml
-spin_manifest_version = "1"
+spin_manifest_version = "2"
```

Comment:
```suggestion
spin_manifest_version = 2
```
(Value now [expected to be an int.](https://github.com/fermyon/spin/blob/main/docs/content/sips/005-manifest-redesign.md#manifest-version))